test/features/insertIf.test.tsx

```diff
@@ -137,6 +137,26 @@ describe('insertIf', () => {
     expect(none.html()).not.toBe(null);
   });
 
+  it('Works with individual CardFields', async () => {
```

Comment:
You can instantiate one Tester with three different fields in its fieldset, then check if those field labels are present, rather than instantiating three testers

Reply:
👍